example_test.go

```diff
@@ -75,17 +75,18 @@ func Example_basicConfiguration() {
 	// See the documentation for Config and zapcore.EncoderConfig for all the
 	// available options.
 	rawJSON := []byte(`{
-		"level": "debug",
-		"encoding": "json",
-		"outputPaths": ["stdout", "/tmp/logs"],
-		"errorOutputPaths": ["stderr"],
-		"initialFields": {"foo": "bar"},
-		"encoderConfig": {
-			"messageKey": "message",
-			"levelKey": "level",
-			"levelEncoder": "lowercase"
-		}
+	  "level": "debug",
+	  "encoding": "json",
+	  "outputPaths": ["stdout", "/tmp/logs"],
+	  "errorOutputPaths": ["stderr"],
+	  "initialFields": {"foo": "bar"},
+	  "encoderConfig": {
+	    "messageKey": "message",
+	    "levelKey": "level",
+	    "levelEncoder": "lowercase"
+	  }
 	}`)
```

Comment:
This is a little hard to grok in the review view, but it formats the JSON to be indented by a tab (to line up with `rawJSON`) and then two spaces per level of JSON indentation. GoDoc.org displays tabs as eight spaces in docs (I think), which makes this JSON look ridiculous.